### ISSUE TITLE
error in Project References when add new file

### ISSUE THREAD

**Issue by holynewbie** [TypeStrong/fork-ts-checker-webpack-plugin#798]
## Current behavior

when `"composite": true` is in `tsconfig.json` ，add a new file and quote it in somewhere, it issue an error

![image](https://user-images.githubusercontent.com/8347286/219617729-9a5aaf85-c622-491e-a048-3309fe82f35e.png)

## Expected behavior

don't issue an error

### package.json

```json
{
  "name": "webpack-demo",
  "version": "1.0.0",
  "description": "",
  "main": "index.js",
  "scripts": {
    "start": "webpack serve"
  },
  "keywords": [],
  "author": "",
  "license": "ISC",
  "devDependencies": {
    "fork-ts-checker-webpack-plugin": "^7.3.0",
    "ts-loader": "^9.4.2",
    "typescript": "^4.9.5",
    "webpack": "^5.75.0",
    "webpack-cli": "^5.0.1",
    "webpack-dev-server": "^4.11.1"
  }
}
```

### tsconfig.json

```json
{
  "compilerOptions": {
    "composite": true,
    "target": "es2016",
    "module": "commonjs",
    "esModuleInterop": true,
    "forceConsistentCasingInFileNames": true,
    "strict": true,
  }
}
```

### webpack.config.js

```js
const ForkTsCheckerWebpackPlugin = require('fork-ts-checker-webpack-plugin');

module.exports = {
    mode: 'development',
    context: __dirname,
    entry: './src/index.ts',
    resolve: {
        extensions: [".ts", ".tsx", ".js"],
    },
    module: {
        rules: [
            {
                test: /\.tsx?$/,
                loader: 'ts-loader',
            }
        ]
    },
    plugins: [new ForkTsCheckerWebpackPlugin()],
    watchOptions: {
        ignored: /node_modules/,
    },
};
```

**start the app!**, and add a new file, quote it in somewhere.

https://user-images.githubusercontent.com/8347286/219620910-70d06703-fc6f-4b58-b7e7-607aa81281a7.mp4

## Environment
 - **fork-ts-checker-webpack-plugin**: [version from the `package.json`]
 - **typescript**: [version from the `package.json`]
 - **eslint**: [version from the `package.json`]
 - **webpack**: [version from the `package.json`]

**Comment by piotr-oles** [comment]
Thanks for the report - it looks like this issue appears only with newer versions on TypeScript. I can't promise I will take a look at this soon but it's on my radar.
